components/sidebar/sidebar-accordion-mui.tsx

```diff
@@ -42,67 +42,72 @@ export const SidebarAccordionMui: FC<SidebarAccordionProps> = ({
 		[setOpenIds]
 	);
 
+	const customComponents: { [key: string]: any } = {
+		create_group: buttonCreateGroup,
```

Comment:
This changes are needed only for Calendars, so they shouldn't be in the Commons submodule.
I suggest to clone and modify the whole component inside the Calendars module